gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/ManagementApplication.java

```diff
@@ -55,6 +48,7 @@ public ManagementApplication() {
         register(ValidationExceptionMapper.class);
         register(JsonMappingExceptionMapper.class);
         register(WebApplicationExceptionMapper.class);
+        register(JsonParseExceptionMapper.class);
```

Comment:
is it possible to register a default error in case there is no match to these specified?

Reply:
well, there is a ThrowableMapper, but it seems not to catch ex in case of malformed json.

Reply:
its not only the ThrowableMapper case. All mappers put into the response message body the exception message. Should we adjust all of them? wdyt? @leleueri  @mstasgravitee